woeip/apps/air_quality/views.py

```diff
@@ -1,20 +1,22 @@
 import datetime
 import logging
 
+from django.contrib import messages
 from django.contrib.auth import authenticate, login
 from django.shortcuts import redirect, render
+
 from woeip.apps.air_quality import dustrak, forms, models
 from woeip.apps.core.models import User
 
 logger = logging.getLogger(__name__)
 
 
-def home(request):
-    return render(request, 'air_quality/home.html')
+def index(request):
+    return render(request, 'index.html')
 
 
-def upload_dustrak(request):
-    """Upload data for a session collected using the Dustrak air quality device and a separate GPS 
+def files(request):
```

Comment:
Similar to below ``files`` seems to general. ``upload_session`` or ``upload`` is more specific and would be my preference.

Reply:
Thinking about it, a closer replication of standard url naming would be to have the model and then the method. 

So, the view, url, and name would be `upload`, `../files/upload/`, and `file-upload`